ib.c.ib

```diff
@@ -372,30 +389,22 @@ static void help(void)
 	     " -S --stdout    -> output to stdout instead of file\n")
 
 int main(const int argc, char **argv)
-	
 	const char * short_options = "hvVs:o:St"
 	const struct option long_options[] =
-			"help",    no_argument,       0, 'h'
-		
-			"verbose", no_argument,       0, 'v'
-		
-			"version", no_argument,       0, 'V'
-		
-			"spaces",  required_argument, 0, 's'
-		
-			"output",  required_argument, 0, 'o'
-		
-			"stdout",  no_argument,       0, 'S'
-		
-			"tab",     no_argument,       0, 't'
-		
-			0, 0, 0, 0
-	
+		{ "help",    no_argument,       0, 'h' }
+		{ "verbose", no_argument,       0, 'v' }
+		{ "version", no_argument,       0, 'V' }
+		{ "spaces",  required_argument, 0, 's' }
+		{ "output",  required_argument, 0, 'o' }
+		{ "stdout",  no_argument,       0, 'S' }
+		{ "tab",     no_argument,       0, 't' }
+		{ 0, 0, 0, 0 }
+
```

Comment:
Double indentation is tricky…perhaps some character like `*`, `-`, `~`, … at the start of a line like in itemisations in some markup languages? Depending on the indentation as before is somewhat error prone and opaque.